obspy/io/sac/core.py

```diff
@@ -381,7 +381,8 @@ def _internal_read_sac(buf, headonly=False, debug_headers=False, fsize=True,
     :return: A ObsPy Stream object.
     """
     # read SAC file
-    sac = SACTrace.read(buf, headonly=headonly, ascii=False, checksize=fsize)
+    sac = SACTrace.read(buf, headonly=headonly, ascii=False, checksize=fsize,
+                        encoding=kwargs.get('encoding', 'ASCII'))
```

Comment:
This kind of hides the default encoding `ASCII` in the belly of this routine.. I don't think it has a downside to move it up as `def _internal_read_sac(..., encoding='ASCII', **kwargs)`. Even though it's an internal low level function not to be used directly by the user there's no need to hide away things, I think. This feels a bit like security by obscurity.. The docstrings should just state that the `encoding` parameter should not be changed normally and that it might produce an 'illegal' SAC file, if people want to use it then it's on their own doing..